src/db/dal/user.ts

```diff
@@ -114,9 +276,60 @@ export const completeRegistration = async (
     return results[1][0];
 };
 
-export const deleteUser = async (keycloak_id: string): Promise<boolean> => {
-    const deletedCount = await UserModel.destroy({
-        where: { keycloak_id },
+export const updateRolesAndDataUsages = async (): Promise<void> => {
+    const results = await UserModel.findAll();
+
+    results.map(async (user) => {
```

Comment:
Are we sure it works? To my knowledge, a `map` does not "wait" for a callback to return a fulfilled or rejected promise

Reply:
c'est plutot un fire and forget ici car on n'a pas besoin d'avoir le retour, c'est la migration one-shot des mauvaises valeurs de roles et portal usages

Reply:
pour await un map au total il faudrait await Promise.all(map()) en effet mais ici celine veut faire 1 par 1 il n'y a pas de probleme